environment.yml

```diff
@@ -3,18 +3,20 @@ channels:
 - PSLmodels
 - conda-forge
 dependencies:
-- python
+- python>=3.7.7
 - ipython
 - setuptools
 - mkl
```

Comment:
We may want to set `mkl>=2020.2` for some of the `txfunc.py` tests that seem to most strongly depend on the mkl version.  Although the results of those tests still varied across platform even with the same mkl version  (meaning pinning the version doesn't itself guarantee the same results).

Reply:
@jdebacker . OK. That is the version currently in my `ogusa-dev` environment without specifying. So I will add it right now and restart the long full_run pytest.